### PR TITLE
fix StopIteration exception and getting payments as dataframe function

### PR DESCRIPTION
**Pull request by basnijholt**
- handle StopIteration for when count is larger than number of transactions
- add `Payments.fetch_account` classmethod
- add `payments_as_dataframe` function

### CODE REVIEW

**Comment by basnijholt** [comment]
I've also added a `Payments.fetch_account` classmethod, which is useful to create a Payments instance from an `account_id`.

File: bunqexport/export.py [hunk 1]
```diff
@@ -78,7 +83,7 @@ class Payments():
     """
 
     def __init__(self, payments):
-        self.payments = pandas.io.json.json_normalize(json.loads(payments))
+        self.payments = pandas.json_normalize(json.loads(payments))
```
Comment (basnijholt) [inline bunqexport/export.py:85]: This is because `pandas.io.json.json_normalize` will be deprecated.

**Comment by basnijholt** [comment]
Finally, I have added a function `payments_as_dataframe`

Which can be used like:
```python
from bunqexport.export import payments_as_dataframe

df = payments_as_dataframe("../bunq-production.conf")
```

**Comment by stuertz** [comment]
Hey, maybe you can also fix existing and add some more tests? 
Please check `make tests`

**Comment by basnijholt** [comment]
I have fixed the tests, even though the failures were unrelated to the changes I made.

I believe that they were failing because pandas slightly changed how it replaces strings.

For example, before the column, `description` had surrounding double quotes and it still included `\n` even though there is a line in the code that was supposed to remove those newlines.

**Comment by stuertz** [comment]
@basnijholt: Thanks again!

**Pull request merged**